### PR TITLE
updated test passes all the cases

### LINKED ISSUE #64: Update the test cases with designed schema
**Describe the enhancement you'd like**
Update the test cases with designed schema, As of now some previous test cases are failing because of updated code.

**Describe approaches if you have thought of any**
Update the unit tests.

Fails: https://travis-ci.org/codeuino/social-platform-donut-backend/builds/657254341?utm_source=github_status&utm_medium=notification

### PR DESCRIPTION
**Pull request by Rupeshiya**
Fixes #64 
Passes all the test cases.
**Screenshot**
testCases
@devesh-verma @vaibhavdaren Please review

### CODE REVIEW

File: test/user.test.js [hunk 1]
```diff
@@ -4,30 +4,54 @@ const jwt = require('jsonwebtoken')
...
   _id: testUserId,
-  name: 'test user',
-  email: 'testuser@mailinator.com',
-  password: 'testUser@123',
-  about: 'I serve to test.',
-  company: 'Codeuino',
-  website: 'http://www.codeuino.com',
-  location: 'Github',
+  ...demoUser,
+  email: 'test@test.com',
```
Comment (devesh-verma) [inline test/user.test.js]: @mailinator.com

File: test/user.test.js [hunk 2]
```diff
@@ -4,30 +4,54 @@ const jwt = require('jsonwebtoken')
 const request = require('supertest')
 const User = require('../app/models/User')
 
+const demoUser = {
+  name: {
+    firstName: 'Rupesh',
```
[the remark this answers is not in the document]
Comment (Rupeshiya) [reply test/user.test.js]: done!

File: test/user.test.js (hunk 1 above)
Comment (Rupeshiya) [reply test/user.test.js]: done!

**Comment by devesh-verma** [comment]
@Rupeshiya good job 👍 
Keep up the good work.

**Pull request merged**